textlint.md

```diff
@@ -1,4 +1,4 @@
-# textlint
+# textlint [![Build Status](https://travis-ci.org/assout/textlint-sandbox.svg?branch=master)](https://travis-ci.org/assout/textlint-sandbox)
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
"textlint [![Build Status](https://travis-ci.org/assout/textlint-sandbox.svg?branch=master)](https://travis-ci.org/assout/textlint-sandbox)" is over 32 (max-length-of-title)